pylib/base/exec_utils.py

```diff
@@ -91,6 +91,34 @@ def ExecuteParallel(args, pool_size=0, callback=PicklableCallback()):
     sys.stdout.flush()
     return []
 
+  @staticmethod
+  def ExecuteSequentially(args, pool_size=0, callback=PicklableCallback()):
+    """Executes a list of methods sequentially. Uses the PicklableCallback
+    as default callback to run individual handlers with the given set of args.
+
+    Args:
+      callback: callable method: A method that can be pickled and called.
+      args: list: List of args to pass to PicklableCallback which in turn calls
+          the callback handler.
+      pool_size: int: The size of the task pool.
+    Return:
+      list: Returns a list of results passed on by the callback handler.
+    """
+    if not args:
+      TermColor.Warning('Nothing to execute.')
+      return []
```

Comment:
`return []` is fine - I was thinking maybe `return map(None, [])` to have a consistent return type - but that's ugly.

I'm also a fan of having a single return type - but again - this is fine.